nobl9/README.md

```diff
@@ -0,0 +1,52 @@
+# Nobl9
+
+## Overview
+Nobl9 is an SLO platform that provides real-time and historical SLO reports.
+Nobl9 integrates with Datadog to collect SLI metrics and measure them against
```

Comment:
Can you list these metrics in the `metadata.csv`?

Reply:
@sarah-witt per @coignetp 's comment [here](https://github.com/DataDog/integrations-extras/pull/1019#discussion_r736232740) it sounds like its not needed.